bool open_gripper
bool cut
bool release_box
---
bool success
